core/domain/suggestion_services.py

```diff
@@ -2056,38 +2069,44 @@ def _update_translation_contribution_stats_models(
             stat.rejected_translation_word_count)
         stats_model.contribution_dates = stat.contribution_dates
 
-    suggestion_models.TranslationContributionStatsModel.update_timestamps_multi(  # type: ignore[type-var]
+    assert stats_models_to_update is not None
```

Comment:
> wait!, In line 2048 you are defining stats_models_to_update as an empty list and at line 2073 your passing this (stats_models_to_update) list directly without appending anything to this list in between. Basically, you are passing empty list to update_timestamps_multi and put_multi methods? -- please correct me if I'm wrong here.

Thanks for pointing it out. I appended `stats_models_to_update` `stats_model`.

> Also, since you are defining stats_models_to_update as a List[suggestion_models.TranslationContributionStatsModel] I really doubt that you need to assert here -- can you please provide the log error that MyPy is saying?

Yeah I removed the assertion.